### DISCUSSION TITLE
Netbox-Docker for Netbox 3.4

### DISCUSSION THREAD

**Discussion by Delta1977** [netbox-community/netbox-docker#898] [General]
Hi, 
when will netbox-docker for netbox 3.4 arive?

**Comment by tobiasge** [comment]
Images were pushed 5 hours ago:
https://hub.docker.com/r/netboxcommunity/netbox/tags?page=1&name=v3.4

**Comment by Delta1977** [comment]
I know,  but Docker-compose (release and Develop) refers to netbox 3.3
https://github.com/netbox-community/netbox-docker/blob/8b16b16e45ca88506521ca03de9320d07ec8e9ed/docker-compose.yml#L4
sure I can adjust them manually, but I don't want to deviate too far from the git repo

**Reply by kkthxbye-code** [reply]
You are supposed either use a docker-compose.override.yml file to set the image version or set the environment variable VERSION.

https://github.com/netbox-community/netbox-docker/wiki/Version

docker compose automatically loads the file called `.env` in the root directory, so you can create a file called `.env` with the following:

```
VERSION=v3.4-2.3.0
```

**Comment by Delta1977** [comment]
Thanks to @tobiasge for build new release:
PR: https://github.com/netbox-community/netbox-docker/pull/899
